scripts/create_narrative.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+import argparse
+import datetime
+import sys
+
+from jinja2 import Environment, FileSystemLoader, select_autoescape
+import pandas as pd
```

Comment:
I see you added `jinja2`, but should probably add `pandas` to the `workflow/envs/nextstrain.yaml` file too?

Reply:
Fair point. I added jinja2 because it isn't otherwise available as a dependency of other dependencies, but pandas is a dep of augur, so I've never explicitly included it here.

Reply:
Following my own logic above, I just removed IQ-TREE and mafft dependencies from the environment, too, and bumped Augur and Nextclade versions to latest. Thanks, @genehack!